install_tpl.sh

```diff
@@ -10,6 +10,7 @@
 
 TMP_DIR="/tmp/install-fossa-cli"
 
+
```

Comment:
[nit] Extra whitespace.

Reply:
It's intentional, because a line will be added at line 12 when each new file is generated.

Reply:
You should add a comment indicating this. Another option might be e.g. a "REPLACE ME" comment that you use as a replacement target for `sed`.